rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-nodes-machinepools-about"]
+= About machine pools
+:context: rosa-nodes-machinepools-about
+
+toc::[]
+
+{product-title} uses machine pools as an elastic, dynamic provisioning method on top of your cloud infrastructure.
+
+The primary resources are machines, machine sets, and machine pools.
+
+== Machines
```

Comment:
This sounds like a concept module.

And the headings in this topic should have anchors.